openapi.json

```diff
@@ -319,6 +319,137 @@
         ]
       }
     },
+    "/clusters/{clusterId}/report/ruleId": {
```

Comment:
This endpoint doesn't follow API design guidelines. It should look like this:

`"/clusters/{clusterId}/rules/{ruleId}report":`

Reply:
updated